lib/google_scraping/dashboard/keyword_scraper.ex

```diff
@@ -0,0 +1,20 @@
+defmodule GoogleScraping.Dashboard.KeywordScraper do
+  @base_url "https://www.google.com/search"
+  @user_agent "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36"
+
+  @doc """
+  Returns the HTML of the Google search page for the given keyword.
+  """
+  def get_search_page_html_for_keyword(keyword) do
+    url =
```

Comment:
Can we use a separate private function to build the url? so that `get_search_page_html_for_keyword` looks ,ore redable

```suggestion
    url = build_url(..)
```

Reply:
Thank you for the refactoring suggestions, all of them available here: https://github.com/Nihisil/nimble-certification-elixir/pull/41/commits/00b304c93f6db7488bb1ed91d06450c904c91caa

It makes the code looks better.